src/Tokenizer/Token.php

```diff
@@ -45,7 +45,7 @@ final class Token
     private bool $changed = false;
 
     /**
-     * @param array{int, string}|string $token token prototype
+     * @param array{int, non-empty-string}|string $token token prototype
```

Comment:
funny we provide empty string only in `Tokens::clearAt()`

Reply:
good improvement!

Reply:
but please fix the new phpstan errors as well

Reply:
let's take as separated PR then

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8242